src/neuroconv/datainterfaces/ecephys/openephys/openephyslegacydatainterface.py

```diff
@@ -34,28 +41,22 @@ def __init__(
         ----------
         folder_path : FolderPathType
             Path to OpenEphys directory.
-        stream_name : str, default: "Signals CH"
+        stream_name : str, optional
             The name of the recording stream.
         verbose : bool, default: True
         """
-
         available_streams = self.get_stream_names(folder_path=folder_path)
-        assert (
-            stream_name in available_streams
-        ), f"The selected stream '{stream_name}' is not in the available streams '{available_streams}'!"
+        if len(available_streams) > 1 and stream_name is None:
+            raise ValueError(
+                "More than one stream is detected! Please specify which stream you wish to load with the `stream_name` argument. "
+                "To see what streams are available, call `OpenEphysRecordingInterface.get_stream_names(folder_path=...)`."
+            )
```

Comment:
Can you add a test for this assertion as well?

Reply:
I would love to, but unfortunately none of the GIN data for the legacy has multiple streams, and I don't know how to mimic it in the extractor either =\

Reply:
I’m not sure how to mimic it in the extractor, I guess we can go with this. Thanks for improving the interface!